internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -151,15 +156,19 @@ func (s *dockerComposeDeployedService) TearDown() error {
 		return errors.Wrap(err, "could not create Docker Compose project for service")
 	}
 
-	downOptions := compose.CommandOptions{
+	containerLogs, err := p.Logs(compose.CommandOptions{
```

Comment:
Having the whole logs in memory can be a lot in some cases, would it be possible to pipe them and receive here a reader?